src/jit/compiler.h

```diff
@@ -598,6 +598,20 @@ class  LclVarDsc
         return (unsigned)(roundUp(lvExactSize, sizeof(void*)));
     }
 
+    bool                lvIsMultiregStruct()
+    {
+#if FEATURE_MULTIREG_ARGS_OR_RET
+#ifdef _TARGET_ARM64_
+        if ((TypeGet() == TYP_STRUCT) &&
+            (lvSize()  == 2 * TARGET_POINTER_SIZE))
+        {
+            return true;
+        }
+#endif  // _TARGET_ARM64_
+#endif  // FEATURE_MULTIREG_ARGS_OR_RET
```

Comment:
Pointers are 8 bytes on all of our 64-bit targets: Arm64 and x64
They are 4 bytes on all of out 32-bit targets:  Arm32 and x86 
It is a future goal to be able to cross compile from a 64-bit Host to a 32-bit Target
So we try to use the TARGET term whenever it is appropriate.